### DISCUSSION TITLE
REGISTRATION PAGE

### DISCUSSION THREAD

**Discussion by djmakers** [rejetto/hfs#538] [Q&A]
### I need the user registration page, once you register, a folder with the user's name is created on the HDD, where only he can have access, is this function possible? Please a lot!

**For example my username is PATRICK**
![image](https://github.com/rejetto/hfs/assets/47154419/56c4f4af-4d83-4b76-bb8b-f14c4fcbcdeb)

I need this for simple users, without adding it myself, and without choosing what to see and what not on the server, the idea would be welcome

**Comment by rejetto** [comment]
HFS at the moment doesn't have anything like that.
it's interesting, but it's hard to satisfy: you need to create a folder with the username, another admin will need something different.
I will consider it but after i'm done with the rest, so don't wait for it. It may happen in a year.
if you are a programmer you may attempt doing that as a plugin.
